eng/native/configurecompiler.cmake

```diff
@@ -502,6 +502,13 @@ if (MSVC)
   add_compile_options($<$<COMPILE_LANGUAGE:C,CXX>:/Zc:inline>) # All inline functions must have their definition available in the current translation unit.
   add_compile_options($<$<COMPILE_LANGUAGE:C,CXX>:/Zc:forScope>) # Enforce standards-compliant for scope.
 
+  add_compile_options($<$<COMPILE_LANGUAGE:C,CXX>:/wd4065>)
```

Comment:
what is the difference between this block and the block below `# Disable Warnings:`?

Reply:
I do not know.